meetings/agenda.md

```diff
@@ -8,3 +8,4 @@ Please file pull requests to add, or discuss items to add, to the agenda.
 ## Items to Discuss
 
 - Whether to add Slamdunk and settermjd as laminas-mime and laminas-mail mainteiners or to mark laminas-mail as a security only package.
+- Whether to abandon [laminas/laminas-pimple-config](https://github.com/laminas/laminas-pimple-config) - Pimple last had a [release in October 2021](https://github.com/silexphp/Pimple/tags), Package stats for [laminas-pimple-config](https://packagist.org/packages/laminas/laminas-pimple-config/stats).
```

Comment:
Pimple is closed for changes: https://github.com/silexphp/Pimple#pimple
Therefore, there are no new releases. We call it "security-only maintenance mode".